app/models/concerns/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+end
```

Comment:
Great job @Zilola-Nazarova 🤝

- [ ] As stated in the project requirements, kindly ensure the comment model has a method that updates the comments counter for a post.
![update-comments-counter](https://github.com/Zilola-Nazarova/blog-app/assets/63917725/e3b7c102-110e-4119-9be4-9cbfe8393051)